app/src/main/java/org/mozilla/fenix/share/ShareController.kt

```diff
@@ -204,7 +204,9 @@ class DefaultShareController(
     }
 
     @VisibleForTesting
-    internal fun getShareSubject() = shareSubject ?: shareData.map { it.title }.joinToString(", ")
+    internal fun getShareSubject() =
+        shareSubject ?: shareData.filterNot { it.title.isNullOrEmpty() }
+            .joinToString(", ") { it.title.orEmpty() }
```

Comment:
```suggestion
            .joinToString() { it.title.orEmpty() }
```
Could leave as is or just remove it since ", " is already the default value.

Reply:
For the separator value, I prefer to have the default value explicitly as the parameter for methods where it is not very obvious that there is a parameter and what is the default value for it.
As for the transform function, we cannot simply remove it, because we want to map the ShareData objects to their titles. So at least `{ it.title } `should be used. And as the title is a `CharSequence` an additional method is needed to make it a `String`. I chose `{ it.title.orEmpty() }` but I can also use `{ it.title.toString() }` if you prefer it.

Reply:
{ it.title.toString() } or { it.title } sounds good to me